project-base/app/translations/customerValidators.cs.po

```diff
@@ -0,0 +1,33 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: cs\n"
+
+msgid "An array of values is not supported as an input for \"slider\" type parameter. Use \"minimalValue\" and \"maximalValue\" instead."
+msgstr "Pole hodnot (\"values\") není podporováno jako vstup pro parametry typu \"slider\". Namísto toho použijte \"minimalValue\" and \"maximalValue\"."
```

Comment:
off topic, I would rather use parameters here to avoid mis-translation of "slider", "minimalValue", and "maximalValue"

Reply:
what about 9b7b9c4eef?